<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressBook</class>
 <widget class="QMainWindow" name="AddressBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Książka adresowa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="searchbox">
        <property name="text">
         <string>Szukaj...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="contact_list_view">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>55</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="new_contact_button">
          <property name="text">
           <string>Nowy kontakt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_contact_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edytuj kontakt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_contact_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Usuń kontakt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quit_button">
          <property name="text">
           <string>Zamknij</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
